torch/_inductor/ir.py

```diff
@@ -5321,6 +5323,22 @@ def codegen(self, wrapper):
 
         # Call to kernel
         self.codegen_comment(wrapper)
+
+        """
+        Filter out None args.
+
+        see _precompile_config(...) and https://github.com/pytorch/pytorch/issues/115344
+
+        Two cases for a None arg:
+        1. The arg is already tl.constexpr, so leave it in
+        2. The arg is not tl.constexpr so we have to remove it
+        """
+        constexpr_args_set = set(constexpr_indices)
+        raw_args = [
+            arg
+            for idx, arg in enumerate(raw_args)
+            if (arg is not None) or (arg is None and idx in constexpr_args_set)
```

Comment:
`raw_args` can contain constexpr args. The logic here is:

1. We want to include any argument that isn't `None` (lhs)
2. If the arg is `None`, check if it is a constexpr---if so leave it in (rhs)

The non-`None` constexprs from raw_args should continue to be included here because they would be captured by the lhs of the statement (```arg is not None```).